pgagroal.spec

```diff
@@ -108,6 +108,7 @@ cd %{buildroot}%{_libdir}/
 %{_docdir}/%{name}/README.md
 %{_docdir}/%{name}/RPM.md
 %{_docdir}/%{name}/SECURITY.md
+%{_docdir}/%{name}/DISTRIBUTIONS.md
```

Comment:
You are missing the `%{__install}` command

Reply:
Fixed

Reply:
Please, force push, as I don't see the changes

Reply:
> Please, force push, as I don't see the changes

See 1e6408d, my fault, now it should contains all the changes.